mapit-backend/src/Repository/UserRepository.php

```diff
@@ -18,6 +18,15 @@ public function __construct(ManagerRegistry $registry)
     {
         parent::__construct($registry, User::class);
     }
+    public function createUser(string $name, string $email, string $password): User {
+        $user = new User();
```

Comment:
I think instead of creating a new User, you could use the Dependency Injection at this point which would reduce the code a bit. Than you would only need line 26-29 at the end.